README.md

```diff
@@ -34,7 +34,7 @@ different configuration files per cluster instance. Tools like PM2 set the
 
 `Hostname` Is matched against the machine host name.
 
-For more information on configuration see [the wiki][2].
+For more information on configuration see [this][2].
```

Comment:
Changed that, since initially I was confused, was thinking it should be in `node-server-skeleton` wiki